service/controller/v17/version_bundle.go

```diff
@@ -12,6 +12,11 @@ func VersionBundle() versionbundle.Bundle {
 				Description: "Add security group descriptions for master nodes.",
 				Kind:        versionbundle.KindChanged,
 			},
+			{
+				Component:   "cloudconfig",
+				Description: "Removed CoreDNS related components (will be managed by chart-operator).",
```

Comment:
Nit: I would skip `will be`. They should be already managed at the time we release this version.

Reply:
To me its not clear without the will be but `(now managed by chart-operator)' is shorter and clear.

Reply:
As I see it's only done in v15.

Reply:
Good catch, fixed.